.github/workflows/codecoverage.yml

```diff
@@ -0,0 +1,57 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+on:
+  push:
+    branches:
+        - master
```

Comment:
Is the indentation wrong here?  Something seems to preventing a match.

Reply:
Extra spaces should not matter.  It looks correct, so IDK why it's not working.

Reply:
I've just rebased and repushed, an the coverage job seems to have run fine for me.

Reply:
Can't see a problem now, though the checks run this time have been the `pull_request` checks rather than the `push` checks.  I'll approve and hopefully will see some checks run on 'push' from the merge.

Oh, I've just realized why the code coverage wasn't run - you're pushing to the `task/coverage-separate` branch, not `master` - it's only supposed to run on pushes to `master`.  D'oh, sorry.

Reply:
The CI should have run for the `pull_request` trigger, though.

Reply:
> The CI should have run for the `pull_request` trigger, though.

Strange.  When I reviewed last night, it definitely showed 12 succussful checks on 'push', rather than 13 on 'pull_request'.  I wouldn't have spotted this potential problem otherwise.